R/wateravailability.R

```diff
@@ -0,0 +1,91 @@
+#' Calculate the Water Stress Index
+#' 
+#' This function calculates the Water Stress Index (estimating the yield depression as a function of water deficiency or surplus)
+#' source is the HELP-framework (STOWA, 2005)
```

Comment:
References can be included by `@references`

Reply:
good point. added